manager/allocator/networkallocator/networkallocator.go

```diff
@@ -706,7 +759,7 @@ func (na *NetworkAllocator) allocateDriverState(n *api.Network) error {
 }
 
 // Resolve network driver
-func (na *NetworkAllocator) resolveDriver(n *api.Network) (driverapi.Driver, string, error) {
+func (na *NetworkAllocator) resolveDriver(n *api.Network) (driverapi.Driver, *driverapi.Capability, string, error) {
```

Comment:
The number of return values is getting too high. Maybe this should be split into one function that returns the driver name, and a second that returns the driver and its capabilities based on that name.